src/ziggurat/messaging/producer.clj

```diff
@@ -214,17 +206,17 @@
         dead-letter-exchange-name (prefixed-queue-name topic-entity dead-letter-exchange)]
     (create-and-bind-queue queue-name exchange-name dead-letter-exchange-name)))
 
-(defn- make-delay-queue-with-retry-count [topic-entity retry-count exponential-backoff-count]
+(defn- make-delay-queue-with-retry-count [topic-entity retry-count]
   (let [{:keys [queue-name exchange-name dead-letter-exchange]} (:delay (rabbitmq-config))
         queue-name                (delay-queue-name topic-entity queue-name)
         exchange-name             (prefixed-queue-name topic-entity exchange-name)
         dead-letter-exchange-name (prefixed-queue-name topic-entity dead-letter-exchange)
-        sequence                  (if (<= retry-count exponential-backoff-count) (inc retry-count) (inc exponential-backoff-count))]
+        sequence                  (inc retry-count)]
```

Comment:
if we want to have 50 retry, do we create 50 different channel?

Reply:
@thediantoni you can just specify [:channels :channel_name :retry :count] as 50. Does that answer your question?

Reply:
is there any concern if we create to many queue?

Reply:
Okay. Did you mean 50 different queues for 50 retry-count? Thanks for letting us know of this case. 

Have to read up on RabbitMQ best practices to come up with a good number of limiting the number queues.

Also, this feature is still in alpha. So, we can make the necessary changes before making this stable.